server.py

```diff
@@ -360,6 +360,41 @@ def healthcheck():
         return jsonify({'status': 'OK'})
 
 
+def get_all_comments_by_survey_id(survey_id):
+    # collect survey based on survey id and and 146 code
+
+    records = db.session.query(SurveyResponse).filter(SurveyResponse.data['survey_id'].astext == survey_id).all()
+    logger.info("Comments retrieved " + str(len(records)))
```

Comment:
Again, can the comments contain sensitive data? if so we shouldn't be logging them

Reply:
no it s a len , i am just counting .